### PR TITLE
fix(build): make proto-update-deps

### PR DESCRIPTION
**Pull request by tbruyelle**
# Description

The URL to the cosmos-sdk `SDK_PROTO_URL` was using a branch that doesn't exists (any more I presume). As a result, `make proto-update-deps` wasn't working properly and was filling all the cosmos proto files with `404 Not Found`.

Fix by using the correct branch name, which is `interchain-security-rebase.0.45.11`.

## Linked issues

No issue

## Type of change

If you've checked more than one of the first three boxes, consider splitting this PR into multiple PRs!

- [ ] `Feature`: Changes and/or adds code behavior, irrelevant to bug fixes
- [x] `Fix`: Changes and/or adds code behavior, specifically to fix a bug
- [ ] `Refactor`: Changes existing code style, naming, structure, etc.
- [ ] `Testing`: Adds testing
- [ ] `Docs`: Adds documentation

## Regression tests

If `Refactor`, describe the new or existing tests that verify no behavior was changed or added where refactors were introduced.

## New behavior tests

If `Feature` or `Fix`, describe the new or existing tests that verify the new behavior is correct and expected.

## Versioning Implications

- [ ] This PR will affect [semantic versioning as defined for ICS](../CONTRIBUTING.md#semantic-versioning)

If the above box is checked, which version should be bumped?

- [ ] `MAJOR`: Consensus breaking changes to both the provider and consumers(s), including updates/breaking changes to IBC communication between provider and consumer(s)
- [ ] `MINOR`: Consensus breaking changes which affect either only the provider or only the consumer(s)
- [x] `PATCH`: Non consensus breaking changes

## Targeting

Please select one of the following:

- [ ] This PR is only relevant to main
- [ ] This PR is relevant to main, and should also be back-ported to ____ (ex: v1.0.0 and v1.1.0)
- [ ] This PR is only relevant to ____ (ex: v1.0.0, v1.1.0, and v1.2.0)

### CODE REVIEW

**Review by smarshall-spitzbart — APPROVED** [review]
Thanks!

**Comment by smarshall-spitzbart** [comment]
@tbruyelle does `make proto-update-deps` work for you on this branch? Still having troubles when I try it myself

**Comment by tbruyelle** [comment]
> @tbruyelle does `make proto-update-deps` work for you on this branch? Still having troubles when I try it myself

What's your troubles ? On my side when I run this command from this branch, I got no error and no diff, which is fine because the third_party folder is up to date.

**Comment by tbruyelle** [comment]
@sainoe sure it's better, done here [2259353](https://github.com/cosmos/interchain-security/pull/830/commits/22593536a92eef4bfa522e0c53bce91ba42a3b85)

Note that this could be improved by fetching the dependency version from the go.mod, thanks to `go list`.

```sh
$ go list -f '{{if .Replace}}{{.Replace.Version}}{{else}}{{.Version}}{{end}}' -m github.com/cosmos/cosmos-sdk
v0.45.13-ics
```

We could do that inside the Makefile to avoid having this hard coded value, what do you think ?

Same apply for other proto dependencies, where I noticed a couple of other potential problems : tendermint and confio/ics23 are outdated, they fetch respectively v0.34.5 and v0.7.1 version of the proto files, while the dependency version in te go.mod is v0.34.26 and v0.9.0.

I can fix that in an other PR.

File: Makefile [hunk 1]
```diff
@@ -118,7 +118,7 @@ TM_URL              = https://raw.githubusercontent.com/tendermint/tendermint/v0
 GOGO_PROTO_URL      = https://raw.githubusercontent.com/regen-network/protobuf/cosmos
 CONFIO_URL          = https://raw.githubusercontent.com/confio/ics23/v0.7.1
 COSMOS_PROTO_URL    = https://raw.githubusercontent.com/regen-network/cosmos-proto/master
-SDK_PROTO_URL 		= https://raw.githubusercontent.com/cosmos/cosmos-sdk/interchain-security-rebase/proto/cosmos
+SDK_PROTO_URL 		= https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.13-ics/proto/cosmos
```
Comment (smarshall-spitzbart) [inline Makefile:121]: `v0.45.15` is more up to date but it shouldn't matter, no proto changes in that version

**Review by MSalopek — APPROVED** [review]
Thank you for making this contribution!

This will work for now.

`third_party` is handled a bit differently in SDK > v0.45 so this issue should not show up later.

**Comment by smarshall-spitzbart** [comment]
@tbruyelle we're happy to merge this, just need a rebase. Thanks

**Comment by tbruyelle** [comment]
> @tbruyelle we're happy to merge this, just need a rebase. Thanks

OK I did a merge since there was already a merge from `main` in this branch.

**Pull request merged**